README.md

```diff
@@ -1 +1,25 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Open-source
+- no ads, no trackers and no third party integrations which are not strictly necessary to make the app work
+- no paywalling for features done locally and don't require an ongoing cost from us
+- Low (or zero) licensing/hosting fees, low architectural complexity
```

Comment:
I this is a bit odd read from a user perspective, "as much client-side code as possible, no dependency on us" is a bit more clear